lib/workerholic/queue.rb

```diff
@@ -1,17 +1,17 @@
 module Workerholic
   # Handles background job enqueueing/dequeuing functionality
   class Queue
-    attr_reader :storage, :name
+    attr_reader :storage, :name, :logger
 
-    def initialize(name = 'workerholic:queue:main')
+    def initialize(name = 'workerholic:queue:main', enable_logger = false)
       @storage = Storage::RedisWrapper.new
       @name = name
-      @logger = LogManager.new
+      @logger = enable_logger ? Logger.new : nil
```

Comment:
agreed, also, not sure this is how we should handle the fact that we don't want to log out anything when running tests. This approach suggests we will need to make this modification on any class using an instance of `LogManager`. I think we should probably find a way to do this globally by defining a test environment.